components/osgi-tests/src/test/java/org/wso2/carbon/analytics/test/osgi/SimulatorAPITestcase.java

```diff
@@ -0,0 +1,316 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.analytics.test.osgi;
+
+import org.ops4j.pax.exam.Configuration;
+import org.ops4j.pax.exam.ExamFactory;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+import org.ops4j.pax.exam.testng.listener.PaxExam;
+import org.osgi.framework.BundleContext;
+import org.testng.Assert;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+import org.wso2.carbon.analytics.test.osgi.util.HTTPResponseMessage;
+import org.wso2.carbon.analytics.test.osgi.util.TestUtil;
+import org.wso2.carbon.container.CarbonContainerFactory;
+import org.wso2.carbon.kernel.CarbonServerInfo;
+
+import java.io.File;
+import java.net.URI;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import javax.inject.Inject;
+
+import static org.wso2.carbon.container.options.CarbonDistributionOption.carbonDistribution;
+import static org.wso2.carbon.container.options.CarbonDistributionOption.copyFile;
+
+@Listeners(PaxExam.class)
+@ExamReactorStrategy(PerClass.class)
+@ExamFactory(CarbonContainerFactory.class)
+public class SimulatorAPITestcase {
+    private static final org.apache.log4j.Logger logger = org.apache.log4j.Logger.getLogger(SimulatorAPITestcase.class);
+
+    private static final int HTTP_PORT = 9390;
+    private static final String HOSTNAME = "localhost";
+    private URI baseURI = URI.create(String.format("http://%s:%d", HOSTNAME, HTTP_PORT));
+
+    private final String DEFAULT_USER_NAME = "admin";
+    private final String DEFAULT_PASSWORD = "admin";
+
+    private static final String validSingleEventConfig = "{\n" +
```

Comment:
Constant names should be uppercase. Applies for below variables as well

Reply:
Fixed with commit [#9e041b8abf0bee874137cdcbafd38e6bde8a4930](https://github.com/wso2/carbon-analytics/pull/1259/commits/9e041b8abf0bee874137cdcbafd38e6bde8a4930)